parser_with_temp.py

```diff
@@ -247,13 +249,11 @@ def run_parser(username, password):
         username=username, password=password, headers=headers)
 
     start_execution = time()
-    try:
-        remove('temp.txt')
-    except OSError:
-        pass
+    open('temp.txt', 'w').close()
```

Comment:
it's much better to use `with` operator instead.

Reply:
done